core/lib/wallet_creator/src/main.rs

```diff
@@ -0,0 +1,70 @@
+use std::{env};
+
+use zksync::{
+    web3::{
+        types::{H160, H256},
+    },
+    zksync_types::{
+        tx::PackedEthSignature,
+    },
+    Network, RpcProvider, Wallet, WalletCredentials,
+};
+
+use zksync_eth_signer::{PrivateKeySigner};
+
+fn eth_random_account_credentials() -> (H160, H256) {
+    let mut eth_private_key = H256::default();
+    eth_private_key.randomize();
+
+    let address_from_pk = PackedEthSignature::address_from_private_key(&eth_private_key).unwrap();
+
+    (address_from_pk, eth_private_key)
+}
+
+async fn create_new_wallet(address: &str, key: &str) -> Result<Wallet<PrivateKeySigner, RpcProvider>, anyhow::Error>{
+    let mut eth_address:H160;
+    let mut eth_private_key:H256;
+    if(address.is_empty() || key.is_empty()) {
+        (eth_address, eth_private_key) = eth_random_account_credentials();
+    } else {
+        eth_address = ETH_ADDR.parse().unwrap();
+        eth_private_key = ETH_PRIVATE_KEY.parse().unwrap();
```

Comment:
I don't think that the variables `ETH_ADDR` and `ETH_PRIVATE_KEY` are still defined
```suggestion
        eth_address = address.parse().unwrap();
        eth_private_key = key.parse().unwrap();
```

Reply:
Updated